frontend/src/pages/projects/projectSettings/ProjectSettingsPage.tsx

```diff
@@ -17,7 +16,7 @@ const ProjectSettingsPage = (): ReactElement => {
       id={ProjectSectionID.SETTINGS}
     >
       <Stack hasGutter>
-        {biasMetricsAreaAvailable && (
+        {useProjectSettingsTabVisible() && (
```

Comment:
Ideally avoid using hooks inline -- easier to just place them at the top of the page as they are "state" (under the hood or directly)